build/Helix/EnsureMachineState.ps1

```diff
@@ -26,14 +26,29 @@ foreach($procName in $processNamesToStop)
 Write-Host "All processes running after attempting to kill unwanted processes:"
 Get-Process
 
-# Uninstall AppX packages that are known to cause issues with our tests:
-$packagesToUninstall = @("*Skype*", "*Windows.Photos*")
-foreach($pkgName in $packagesToUninstall)
-{
-    foreach($pkg in (Get-AppxPackage $pkgName).PackageFullName) 
+function UninstallApps {
+    Param([string[]]$appsToUninstall)
+
+    foreach($pkgName in $appsToUninstall)
     {
-        Write-Output "Removing: $pkg" 
-        Remove-AppxPackage $pkg
-    } 
+        foreach($pkg in (Get-AppxPackage $pkgName).PackageFullName) 
+        {
+            Write-Output "Removing: $pkg" 
+            Remove-AppxPackage $pkg
+        } 
```

Comment:
Should we do the "-AllUsers" thing here? Or maybe we haven't seen that problem in Helix yet

Reply:
Probably wouldn't hurt. I will make the change.

---
In reply to: [301833339](https://github.com/microsoft/microsoft-ui-xaml/pull/1025#discussion_r301833339) [](ancestors = 301833339)

Reply:
I started hitting PackageManager errors when using -AllUsers, even when running as admin. I am going to leave it as it is for now. If we start hitting per-user install issues we can take another look.

---
In reply to: [301833621](https://github.com/microsoft/microsoft-ui-xaml/pull/1025#discussion_r301833621) [](ancestors = 301833621,301833339)